test/mocks/config/cspReportUriImpl.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+
+module.exports = {
+	reportOnly: true,
+	impl: function (req) {
+		return 'http://www.example.com?token=sometoken';
```

Comment:
Hmm, weird it shows up that way.  Just being consistent w/ the other `test/mocks/config` files.